pylintrc_tweaks

```diff
@@ -34,6 +34,32 @@ disable+ =
     no-name-in-module,
     unnecessary-lambda-assignment,
 
+    unused-import,
+    yield-outside-function,
+    return-outside-function,
+    line-too-long,
+    bare-except,
+    syntax-error,
+    trailing-whitespace,
+    empty-docstring,
+    unnecessary-pass,
+    return-in-init,
+    continue-in-finally,
+    duplicate-bases,
+    unexpected-special-method-signature,
+    invalid-all-object,
+    misplaced-bare-raise,
+    logging-too-many-args,
+    logging-too-few-args,
+    bad-str-strip-call,
+    too-many-statements,
+    too-many-boolean-expressions,
+    useless-else-on-loop,
+    import-self,
+    global-variable-not-assigned,
+    global-at-module-level,
+    binary-op-exception,
```

Comment:
Suggestion: list these in alphabetical order so it's easier to see if a particular check is in the list or not.